### PR TITLE
Add "@slack/socket-mode" to packages list

### PR DESCRIPTION
**Pull request by g12i**
###  Summary

This package was missing in the issue template.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

**Review by seratch — APPROVED** [review]

**Comment by seratch** [comment]
Thanks for taking the time to make this PR. It looks good to me. The CI build failures is obviously unrelated to your changes but other maintainers will be checking on Monday, Pacific Time.

**Comment by mwbrooks** [comment]
Hey @g12i, great catch on the issue template! 👍🏻 

The failing test is definitely not related to this PR (we recently switch to GitHub Actions and have a few small quirks to work out).

Would you mind signing the CLA so that we can merge this PR? It's very quick and easy - [just select the link from the CLAssistant message above](https://github.com/slackapi/node-slack-sdk/pull/1157#issuecomment-762292918). Once it's signed, I'll merge this PR!

**Review by mwbrooks — APPROVED** [review]

**Comment by mwbrooks** [comment]
Thanks again @g12i 👋🏻 Merged!

**Pull request merged**
